### ISSUE TITLE
Get DB parameters from Symfony 4 .env file

### ISSUE THREAD

**Issue by elvismdev** [backup-manager/symfony#35]
Hi again, 

So now I encounter with the issue that in new SF4 the way to define the DB details are in a one line only, and not with each piece (user, pass, host, dbname) being separated like this bundle needs. 

https://symfony.com/doc/current/configuration.html#the-env-file-environment-variables

```
###> doctrine/doctrine-bundle ###
# Format described at http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
# For an SQLite database, use: "sqlite:///%kernel.project_dir%/var/data.db"
# Configure your db driver and server_version in config/packages/doctrine.yaml
DATABASE_URL=mysql://root:root@127.0.0.1:3306/test_db
###< doctrine/doctrine-bundle ###
```

Is there a way `backup-manager` bundle can get/resolve all the database parameters needed from this `DATABASE_URL` environment variable?

**Comment by Nyholm** [comment]
That is an excellent feature. Thank you

I will see if I can make this work

**Comment by Nyholm** [comment]
Thank you for reporting these issues. I've released 2.1.0 now.
